pkg/infra/appcontext/user.go

```diff
@@ -48,6 +53,14 @@ func User(ctx context.Context) (*user.SignedInUser, error) {
 	if ok {
 		for _, group := range k8sUserInfo.GetGroups() {
 			switch group {
+			case k8suser.AllAuthenticated:
```

Comment:
AllAuthenticated??  will need to investigate this a bit more 😏

Reply:
Usually, this group (`system:authenticated`) gets added as a supplemental group to `system:masters` and `system:apiserver`. With the User (Note: _not group_) `system:aggregator`, I saw that the only group assigned was `system:authenticated`. Once added, the discovery logs stopped reporting failures in parent kube-apiserver.